docs/modules/ROOT/pages/cli.adoc

```diff
@@ -0,0 +1,99 @@
+= Hazelcast Cloud CLI
+
+Hazelcast Cloud CLI is known as `hzcloud` is a command line tool to make actions on Hazelcast Cloud easily. Hazelcast Cloud offers the leading in-memory computing platform, Hazelcast IMDG, as a fully managed service that integrates with your existing virtual private cloud.
+
+== Installing `hzcloud`
```

Comment:
```suggestion
== Installing the Hazelcast Viridian CLI
```

I don't think we should use `hzcloud` as the product name. Users will not search Google for this.